push

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+shopt nullglob
+
+OS=${OS:-fedora}
+VERSION=${VERSION:-40}
+OSVER=$OS-$VERSION
+WORKDIR=$PWD/work
+REPOSITORY=quay.io/foreman/nboci-files
+
+# create image index and set the annotation at the index level
+# TODO set the artifactType at the index level https://github.com/containers/podman/issues/23598
+podman manifest create "$REPOSITORY:$OSVER" --annotation org.pulpproject.netboot.version=1
+
+mkdir -p "$WORKDIR"
+
+for containerfile in "Containerfile-$OS"-* ; do
+	ARCH=${containerfile##*-}
+	VOLUME=$(mktemp -d -p "$PWD/work")
+	podman build -v "${VOLUME}:/root:Z" -f "$containerfile" --platform "linux/$ARCH" --build-arg "version=$VERSION"
+
+	# create and add netboot artifacts to the image index
+	podman manifest add --annotation org.pulpproject.netboot.version=1 \
+	    --os linux --arch "$ARCH" --os-version "$OSVER" \
+	    --artifact \
+	    --artifact-type application/vnd.org.pulpproject.netboot.artifact.v1 \
+	    --artifact-layer-type application/x-netboot-file \
+	    "$REPOSITORY:$OSVER-$ARCH" "$VOLUME"/*
```

Comment:
tag here is just $OSVER without arch

Reply:
I copied that from the existing lines:

https://github.com/theforeman/nboci-files/blob/b8ed17a43196b26af61a8dd45aef41a904d74b9d/push-fedora-40.sh#L20
https://github.com/theforeman/nboci-files/blob/b8ed17a43196b26af61a8dd45aef41a904d74b9d/push-fedora-40.sh#L31

Is that a bug in the existing code as well?

Reply:
yes

Reply:
my oversight from previous pr

Reply:
What about the skopeo copy command? Does it also need to drop the suffix?

Reply:
no, that one is correct, because i am tagging the sub-arches with corresponding tag like fedora-40-amd64